docs/coding-guidelines/typescript.md

```diff
@@ -85,13 +86,15 @@ type of data and supplying _what you want_.
 Because TypeScript infers most types don't feel obligated to add types to everthing, especially literal values.
 
 #### Avoid
+
 ```ts
-const name: string = 'Calypso';
-const isDesktopBuild: boolean = false;
```

Comment:
@scinos this change kind of defeats the purpose of the "avoid" example 🙃

Reply:
also, why on earth would a linter mess with the contents inside a fenced code block?

Reply:
Linter can be disabled with the usual `//eslint-disable` and friends. I missed this one :(

The main goal of linting code blocks is to make sure our docs follow our standard, so anybody can copy/paste a piece of code form our doc and it is ready to go. I agree it gets a bit muddy when we have examples of what _not_ to do.

Reply:
do you want me to open a PR to undo the lint breakage or do you have that?

Reply:
I'll fix it, don't worry. Thanks for the offer anyway :)

Reply:
thanks!

Reply:
#49584